### ISSUE TITLE
Review CLI commands to make sure synopses are consistent 

### ISSUE THREAD

**Issue by mairaw** [dotnet/docs#1523]
With PR #1519, for commands that have multiple usage options like dotnet sln and dotnet new, we're going to list them individually in separate lines to show the different usage options per individual command, such as 

```
dotnet sln [--help]
dotnet sln [<solution_file>] add [--help] [<projects>]
dotnet sln [<solution_file>] list [--help] 
etc.
```

This issue is to review all the CLI command pages to make sure we're being consistent about this,

**Comment by mairaw** [comment]
We should also make sure the commands page list all the options according to the latest CLI/SDK version.

**Comment by mairaw** [comment]
Fixed with #1649
